### PR TITLE
west.yml: update zephyr Apr17

### PR DESCRIPTION
**Pull request by kv2019i**
Update Zephyr to new baseline containing following patches affecting SOF targets:

8a63a0a56339 intel_adsp: ipc: Fix policy state lock usage
e85226fd1a03 soc/intel_adsp: ipc: Remove fragile device state check
741bbaca6dd7 soc/intel_adsp: ipc: Remove unnecessary device state lock
a200dd88d878 dts: xtensa: intel_adsp: Set soft-off state as disabled
d5897a48aaab ipc: intel_adsp: Ensure IPC completion before runtime idle

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Comment by kv2019i** [comment]
CI good, waiting for another review to merge.

**Pull request merged**
